docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -233,6 +361,28 @@ Returns the square root of `col`.
 
 ---
 
+### **`TAN`**
+
+```sql title="Since: 0.27.0"
+TAN(col1)
+```
+
+Returns the tangent of `col1`. `col1` is in radians. Use [RADIANS](#radians) to convert the input to radians if necessary.
+
+This implementation returns a large value approaching positive or negative infinity near the asymptotes since π/2 and similar values cannot be represented exactly.
```

Comment:
```suggestion
This implementation returns a large value approaching positive or negative infinity near the asymptotes, because _π/2_ and similar values cannot be represented exactly.
```